### ISSUE TITLE
Remove problematic details tag from doc-site.

### ISSUE THREAD

**Issue by pranav300** [cerner/terra-framework#883]
# Bug Report

## Description

The inert polyfill doesn't make `details` tag go out of `tab order` which causes an accessibility violation in the main doc-site. 

## @ Mentions

@bjankord
